assets/src/dashboard/components/inlineInputForm/index.js

```diff
@@ -97,6 +93,7 @@ const InlineInputForm = ({
         placeholder={placeholder}
         hasError={Boolean(error)}
         hint={error}
+        aria-label={label}
```

Comment:
I just did the same in #5793 trying to get React 17 to work.

You'll also need 55f7f2b9e325b8baf4f2445a97164a1c07686495 to make sure the renaming still works in the editor.

Feel free to borrow some of my work from that PR, or we can create a joint PR with all these fixes.